<!DOCTYPE html>
<html class="writer-html5" lang="en" data-content_root="./">
<head>
  <meta charset="utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>5. Check relatedness (Sample-level) &mdash; Genotype data analysis UMCG-GRIAC 2023 documentation</title>
      <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=80d5e7a1" />
      <link rel="stylesheet" type="text/css" href="_static/css/theme.css?v=19f00094" />
      <link rel="stylesheet" type="text/css" href="_static/css/custom.css?v=83b30c75" />

  
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
        <script src="_static/jquery.js?v=5d32c60e"></script>
        <script src="_static/_sphinx_javascript_frameworks_compat.js?v=2cd50e6c"></script>
        <script src="_static/documentation_options.js?v=a4664895"></script>
        <script src="_static/doctools.js?v=888ff710"></script>
        <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="_static/js/theme.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="6. Hardy–Weinberg equilibrium (SNP level)" href="Hardy--Weinberg_equilibrium.html" />
    <link rel="prev" title="4. Check sex discordance (Sample-level)" href="Check_sex_discordance.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="index.html" class="icon icon-home">
            Genotype data analysis UMCG-GRIAC
          </a>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <p class="caption" role="heading"><span class="caption-text">Genome studio</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="Processing_raw_data.html">Generate plink files</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Quality control</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="QC_overview.html">Quality control overview</a></li>
<li class="toctree-l1"><a class="reference internal" href="Check_call_rate_per_SNP_per_sample.html">Check call rate per SNP and per sample</a></li>
<li class="toctree-l1"><a class="reference internal" href="Check_MAF.html">Check MAF (SNP-level)</a></li>
<li class="toctree-l1"><a class="reference internal" href="Check_sample_heterozygosity.html">Check sample heterozygosity (Sample-level)</a></li>
<li class="toctree-l1"><a class="reference internal" href="Check_sex_discordance.html">Check sex discordance (Sample-level)</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Check relatedness (Sample-level)</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#ld-pruning">LD pruning</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#calculate-ibd-identity-by-descent-the-degree-of-recent-shared-ancestry">Calculate IBD - identity by descent (the degree of recent shared ancestry)</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="Hardy--Weinberg_equilibrium.html">Hardy--Weinberg equilibrium (SNP level)</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">GWAS</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="GWAS.html">GWAS steps</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">Genotype data analysis UMCG-GRIAC</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="index.html" class="icon icon-home" aria-label="Home"></a></li>
      <li class="breadcrumb-item active">5. Check relatedness (Sample-level)</li>
      <li class="wy-breadcrumbs-aside">
            <a href="_sources/Check_relatedness.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section id="check-relatedness-sample-level">
<h1>5. Check relatedness (Sample-level)<a class="headerlink" href="#check-relatedness-sample-level" title="Link to this heading"></a></h1>
<p>Population-based cohort studies are often limited to unrelated individuals, and the generalized linear modeling approach
(used in GWAS) assumes independence across individuals. Importantly, in regional cohort studies (e.g., hospital-based cohort studies) of complex
diseases, individuals from the same family can be recruited unintentionally. A common measure
of relatedness (or duplication) between pairs of samples is based on identity by descent (IBD). An
IBD kinship coefficient of greater than 0.1875 may suggest relatedness, duplicates, or sample mixture.
Typically, the individual of a related pair with lower genotype call rate is removed.</p>
<section id="ld-pruning">
<h2>LD pruning<a class="headerlink" href="#ld-pruning" title="Link to this heading"></a></h2>
<p>First we need to perform linkage disequilibrium (LD) pruning with a threshold value of 0.7 (this could vary, some studies use 0.2) - remove SNPs with high LD with each other (removes one from each pair).
This eliminates a large degree of redundancy in the data.</p>
<section id="calculate-ibd-identity-by-descent-the-degree-of-recent-shared-ancestry">
<h3>Calculate IBD - identity by descent (the degree of recent shared ancestry)<a class="headerlink" href="#calculate-ibd-identity-by-descent-the-degree-of-recent-shared-ancestry" title="Link to this heading"></a></h3>
<p>This dimension reduction step is commonly applied prior to both IBD analysis and PCA.</p>
</section>
</section>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="Check_sex_discordance.html" class="btn btn-neutral float-left" title="4. Check sex discordance (Sample-level)" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
        <a href="Hardy--Weinberg_equilibrium.html" class="btn btn-neutral float-right" title="6. Hardy–Weinberg equilibrium (SNP level)" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2023, Martin.</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>