<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8" /><meta name="generator" content="Docutils 0.18.1: http://docutils.sourceforge.net/" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>4. Check sex discordance (Sample-level) &mdash; Genotype data analysis UMCG-GRIAC 2023 documentation</title>
      <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
      <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
      <link rel="stylesheet" href="_static/css/custom.css" type="text/css" />
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
        <script src="_static/jquery.js?v=5d32c60e"></script>
        <script src="_static/_sphinx_javascript_frameworks_compat.js?v=2cd50e6c"></script>
        <script src="_static/documentation_options.js?v=a4664895"></script>
        <script src="_static/doctools.js?v=888ff710"></script>
        <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="_static/js/theme.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="5. Check relatedness (Sample-level)" href="Check_relatedness.html" />
    <link rel="prev" title="3. Check sample heterozygosity (Sample-level)" href="Check_sample_heterozygosity.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="index.html" class="icon icon-home">
            Genotype data analysis UMCG-GRIAC
          </a>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <p class="caption" role="heading"><span class="caption-text">Genome studio</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="Processing_raw_data.html">Generate plink files</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Quality control</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="QC_overview.html">Quality control overview</a></li>
<li class="toctree-l1"><a class="reference internal" href="Check_call_rate_per_SNP_per_sample.html">Check call rate per SNP and per sample</a></li>
<li class="toctree-l1"><a class="reference internal" href="Check_MAF.html">Check MAF (SNP-level)</a></li>
<li class="toctree-l1"><a class="reference internal" href="Check_sample_heterozygosity.html">Check sample heterozygosity (Sample-level)</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Check sex discordance (Sample-level)</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#check-y-chr-counts-also-if-there-are-strange-samples">Check Y chr counts also if there are strange samples</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="Check_relatedness.html">Check relatedness (Sample-level)</a></li>
<li class="toctree-l1"><a class="reference internal" href="Hardy--Weinberg_equilibrium.html">Hardy--Weinberg equilibrium (SNP level)</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">GWAS</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="GWAS.html">GWAS steps</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">Genotype data analysis UMCG-GRIAC</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="index.html" class="icon icon-home" aria-label="Home"></a></li>
      <li class="breadcrumb-item active">4. Check sex discordance (Sample-level)</li>
      <li class="wy-breadcrumbs-aside">
            <a href="_sources/Check_sex_discordance.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section id="check-sex-discordance-sample-level">
<h1>4. Check sex discordance (Sample-level)<a class="headerlink" href="#check-sex-discordance-sample-level" title="Link to this heading"></a></h1>
<p>Sample mix-up is investigated by looking at sex mismatch, where sex
information of each sample is compared with genotypes at chromosomes X and Y.</p>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p><strong>NB:</strong> This method however does not detect same sex sample mix-ups and is not reliable when there are sex chromosome abnormalities.</p>
</div>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p><strong>NB:</strong> Fot this step check that ped file has recorded sex, otherwise consider using sex from metadata</p>
</div>
<div class="literal-block-wrapper docutils container" id="id1">
<div class="code-block-caption"><span class="caption-text">For this step we need to use files with already excluded low call rate SNPs</span><a class="headerlink" href="#id1" title="Link to this code"></a></div>
<div class="highlight-r notranslate"><div class="highlight"><pre><span></span><span class="nf">runPLINK</span><span class="p">(</span><span class="s">&quot;--ped file.ped --map file.map --mind 0.1 --recode --out file_SNP_filtered&quot;</span><span class="p">)</span>
</pre></div>
</div>
</div>
<div class="literal-block-wrapper docutils container" id="id2">
<div class="code-block-caption"><span class="caption-text">Now let’s check check sex discordance</span><a class="headerlink" href="#id2" title="Link to this code"></a></div>
<div class="highlight-r notranslate"><div class="highlight"><pre><span></span><span class="nf">runPLINK</span><span class="p">(</span><span class="s">&quot;--ped file_SNP_filtered.ped --map file_SNP_filtered.map --check-sex --out file_SNP_filtered&quot;</span><span class="p">)</span>
<span class="nf">system</span><span class="p">(</span><span class="s">&quot;cat file_SNP_filtered.sexcheck | tr -s &#39; &#39; &#39;\t&#39; &gt;  file_SNP_filtered_sexcheck.tabs&quot;</span><span class="p">)</span>
</pre></div>
</div>
</div>
<div class="literal-block-wrapper docutils container" id="id3">
<div class="code-block-caption"><span class="caption-text">Plot reported sex vs F value (X heterozygosity level).</span><a class="headerlink" href="#id3" title="Link to this code"></a></div>
<div class="highlight-r notranslate"><div class="highlight"><pre><span></span><span class="n">Plink_sex_filtered</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="nf">read.delim</span><span class="p">(</span><span class="s">&quot;file_SNP_filtered_sexcheck.tabs&quot;</span><span class="p">)</span><span class="w"> </span><span class="o">%&gt;%</span>
<span class="w">        </span><span class="nf">select</span><span class="p">(</span><span class="o">-</span><span class="n">X</span><span class="p">)</span>
<span class="n">Plink_sex_filtered</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">Plink_sex_filtered</span><span class="w"> </span><span class="o">%&gt;%</span>
<span class="w">  </span><span class="nf">mutate</span><span class="p">(</span><span class="n">PEDSEX</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="nf">if_else</span><span class="p">(</span><span class="n">PEDSEX</span><span class="w"> </span><span class="o">==</span><span class="w"> </span><span class="s">&quot;2&quot;</span><span class="p">,</span><span class="w"> </span><span class="s">&quot;female&quot;</span><span class="p">,</span><span class="w"> </span><span class="s">&quot;male&quot;</span><span class="p">))</span>

<span class="nf">ggplot</span><span class="p">(</span><span class="n">Plink_sex_filtered</span><span class="p">,</span><span class="w"> </span><span class="nf">aes</span><span class="p">(</span><span class="n">x</span><span class="o">=</span><span class="n">PEDSEX</span><span class="p">,</span><span class="w"> </span><span class="n">y</span><span class="o">=</span><span class="w"> </span><span class="n">`F`</span><span class="p">))</span><span class="w"> </span><span class="o">+</span>
<span class="w">        </span><span class="nf">geom_jitter</span><span class="p">(</span><span class="n">color</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="nf">ifelse</span><span class="p">((</span><span class="n">Plink_sex_filtered</span><span class="o">$</span><span class="n">PEDSEX</span><span class="w"> </span><span class="o">==</span><span class="w"> </span><span class="s">&quot;female&quot;</span><span class="p">),</span><span class="w"> </span><span class="s">&quot;red&quot;</span><span class="p">,</span><span class="w"> </span><span class="s">&quot;blue&quot;</span><span class="p">))</span><span class="w"> </span><span class="o">+</span>
<span class="w">        </span><span class="nf">geom_text_repel</span><span class="p">(</span><span class="nf">aes</span><span class="p">(</span><span class="n">label</span><span class="o">=</span><span class="nf">ifelse</span><span class="p">(((</span><span class="n">PEDSEX</span><span class="w"> </span><span class="o">==</span><span class="w"> </span><span class="s">&quot;female&quot;</span><span class="p">)</span><span class="w"> </span><span class="o">&amp;</span><span class="w"> </span><span class="p">(</span><span class="s">&quot;F&quot;</span><span class="w"> </span><span class="o">&gt;</span><span class="w"> </span><span class="m">0.2</span><span class="p">)),</span><span class="w"> </span><span class="n">IID</span><span class="p">,</span><span class="w"> </span><span class="s">&quot;&quot;</span><span class="p">),</span>
<span class="w">                                                </span><span class="n">hjust</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="m">0.4</span><span class="p">,</span><span class="w"> </span><span class="n">vjust</span><span class="o">=</span><span class="w"> </span><span class="m">0.4</span><span class="p">))</span><span class="o">+</span>
<span class="w">        </span><span class="nf">xlab</span><span class="p">(</span><span class="s">&quot;Reported sex&quot;</span><span class="p">)</span><span class="o">+</span>
<span class="w">        </span><span class="nf">ylab</span><span class="p">(</span><span class="s">&quot;Heterozygosity rate&quot;</span><span class="p">)</span><span class="o">+</span>
<span class="w">        </span><span class="nf">theme_bw</span><span class="p">()</span>
</pre></div>
</div>
</div>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>plink report marks problematic samples, this information should be available in file_SNP_filtered_sexcheck.tabs
<strong>Advice</strong> It is also important to look at the plot and asses it visually</p>
</div>
<section id="check-y-chr-counts-also-if-there-are-strange-samples">
<h2>Check Y chr counts also if there are strange samples<a class="headerlink" href="#check-y-chr-counts-also-if-there-are-strange-samples" title="Link to this heading"></a></h2>
<div class="literal-block-wrapper docutils container" id="id4">
<div class="code-block-caption"><span class="caption-text">Plot y chromosome counts</span><a class="headerlink" href="#id4" title="Link to this code"></a></div>
<div class="highlight-r notranslate"><div class="highlight"><pre><span></span><span class="n">Plink_sex_filtered_ycount</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">Plink_sex_filtered_ycount</span><span class="w"> </span><span class="o">%&gt;%</span>
<span class="w">        </span><span class="nf">mutate</span><span class="p">(</span><span class="n">PEDSEX</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="nf">if_else</span><span class="p">(</span><span class="n">PEDSEX</span><span class="w"> </span><span class="o">==</span><span class="w"> </span><span class="s">&quot;2&quot;</span><span class="p">,</span><span class="w"> </span><span class="s">&quot;female&quot;</span><span class="p">,</span><span class="w"> </span><span class="s">&quot;male&quot;</span><span class="p">))</span>

<span class="nf">ggplot</span><span class="p">(</span><span class="n">Plink_sex_filtered_ycount</span><span class="p">,</span><span class="w"> </span><span class="nf">aes</span><span class="p">(</span><span class="n">x</span><span class="o">=</span><span class="n">PEDSEX</span><span class="p">,</span><span class="w"> </span><span class="n">y</span><span class="o">=</span><span class="w"> </span><span class="n">YCOUNT</span><span class="w"> </span><span class="p">))</span><span class="w"> </span><span class="o">+</span>
<span class="w">        </span><span class="nf">geom_text_repel</span><span class="p">(</span><span class="nf">aes</span><span class="p">(</span><span class="n">label</span><span class="o">=</span><span class="nf">ifelse</span><span class="p">(((</span><span class="n">PEDSEX</span><span class="w"> </span><span class="o">==</span><span class="w"> </span><span class="s">&quot;male&quot;</span><span class="p">)</span><span class="w"> </span><span class="o">&amp;</span><span class="w"> </span><span class="p">(</span><span class="n">YCOUNT</span><span class="w"> </span><span class="o">&lt;</span><span class="w"> </span><span class="m">3000</span><span class="p">)),</span><span class="w"> </span><span class="n">IID</span><span class="p">,</span><span class="s">&quot;&quot;</span><span class="p">),</span>
<span class="w">                                        </span><span class="n">hjust</span><span class="o">=</span><span class="w"> </span><span class="m">0.4</span><span class="p">,</span><span class="w"> </span><span class="n">vjust</span><span class="o">=</span><span class="w"> </span><span class="m">0.4</span><span class="p">))</span><span class="w"> </span><span class="o">+</span>
<span class="w">        </span><span class="nf">xlab</span><span class="p">(</span><span class="s">&quot;Reported sex&quot;</span><span class="p">)</span><span class="w"> </span><span class="o">+</span>
<span class="w">        </span><span class="nf">ylab</span><span class="p">(</span><span class="s">&quot;Y counts&quot;</span><span class="p">)</span><span class="w"> </span><span class="o">+</span>
<span class="w">        </span><span class="nf">theme_bw</span><span class="p">()</span>
</pre></div>
</div>
</div>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p><strong>Advice:</strong> Look at the plot and check that males and females samples are clearly separating on Y-axis</p>
</div>
</section>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="Check_sample_heterozygosity.html" class="btn btn-neutral float-left" title="3. Check sample heterozygosity (Sample-level)" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
        <a href="Check_relatedness.html" class="btn btn-neutral float-right" title="5. Check relatedness (Sample-level)" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2023, Martin.</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>