<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8" /><meta name="generator" content="Docutils 0.18.1: http://docutils.sourceforge.net/" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>3. Check sample heterozygosity (Sample-level) &mdash; Genotype data analysis UMCG-GRIAC 2023 documentation</title>
      <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
      <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
      <link rel="stylesheet" href="_static/css/custom.css" type="text/css" />
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
        <script src="_static/jquery.js?v=5d32c60e"></script>
        <script src="_static/_sphinx_javascript_frameworks_compat.js?v=2cd50e6c"></script>
        <script src="_static/documentation_options.js?v=a4664895"></script>
        <script src="_static/doctools.js?v=888ff710"></script>
        <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="_static/js/theme.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="4. Check sex discordance (Sample-level)" href="Check_sex_discordance.html" />
    <link rel="prev" title="2. Check MAF (SNP-level)" href="Check_MAF.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="index.html" class="icon icon-home">
            Genotype data analysis UMCG-GRIAC
          </a>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <p class="caption" role="heading"><span class="caption-text">Genome studio</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="Processing_raw_data.html">Generate plink files</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Quality control</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="QC_overview.html">Quality control overview</a></li>
<li class="toctree-l1"><a class="reference internal" href="Check_call_rate_per_SNP_per_sample.html">Check call rate per SNP and per sample</a></li>
<li class="toctree-l1"><a class="reference internal" href="Check_MAF.html">Check MAF (SNP-level)</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Check sample heterozygosity (Sample-level)</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#plot-sample-call-rate-vs-heterozygosity">Plot sample call rate vs heterozygosity</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="Check_sex_discordance.html">Check sex discordance (Sample-level)</a></li>
<li class="toctree-l1"><a class="reference internal" href="Check_relatedness.html">Check relatedness (Sample-level)</a></li>
<li class="toctree-l1"><a class="reference internal" href="Hardy--Weinberg_equilibrium.html">Hardy--Weinberg equilibrium (SNP level)</a></li>
<li class="toctree-l1"><a class="reference internal" href="PCA.html">PCA (SNP level)</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">Genotype data analysis UMCG-GRIAC</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="index.html" class="icon icon-home" aria-label="Home"></a></li>
      <li class="breadcrumb-item active">3. Check sample heterozygosity (Sample-level)</li>
      <li class="wy-breadcrumbs-aside">
            <a href="_sources/Check_sample_heterozygosity.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section id="check-sample-heterozygosity-sample-level">
<h1>3. Check sample heterozygosity (Sample-level)<a class="headerlink" href="#check-sample-heterozygosity-sample-level" title="Link to this heading"></a></h1>
<p>Heterozygosity refers to the presence of each of the two alleles
at a given SNP within an individual.
Outliers showing excess or depletion in heterozygotes
genotypes may be due to contamination or issues during genotyping process.</p>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>Consider doing it already only with high call rate SNPs?</p>
</div>
<div class="highlight-r notranslate"><div class="highlight"><pre><span></span><span class="nf">runPLINK</span><span class="p">(</span><span class="s">&quot;--ped file.ped --map file.map --het --out file&quot;</span><span class="p">)</span>
<span class="nf">system</span><span class="p">(</span><span class="s">&quot;cat file.het | tr -s &#39; &#39; &#39;\t&#39; &gt; file_het.tabs&quot;</span><span class="p">)</span>
<span class="n">het_level</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="nf">read.delim </span><span class="p">(</span><span class="s">&quot;file_het.tabs&quot;</span><span class="p">)</span><span class="w"> </span><span class="o">%&gt;%</span>
<span class="w">  </span><span class="nf">select </span><span class="p">(</span><span class="o">-</span><span class="n">X</span><span class="p">)</span>
</pre></div>
</div>
<section id="plot-sample-call-rate-vs-heterozygosity">
<h2>Plot sample call rate vs heterozygosity<a class="headerlink" href="#plot-sample-call-rate-vs-heterozygosity" title="Link to this heading"></a></h2>
<div class="highlight-r notranslate"><div class="highlight"><pre><span></span><span class="n">sample_missing_rate_het</span><span class="w"> </span><span class="o">&lt;-</span><span class="w"> </span><span class="n">call_rate_samples_after_filter</span><span class="w"> </span><span class="o">%&gt;%</span>
<span class="w">        </span><span class="nf">left_join</span><span class="p">(</span><span class="n">het_level</span><span class="p">,</span><span class="w"> </span><span class="n">by</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="nf">c</span><span class="p">(</span><span class="s">&quot;IID&quot;</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="s">&quot;IID&quot;</span><span class="p">))</span><span class="w"> </span><span class="o">%&gt;%</span>
<span class="w">        </span><span class="nf">rename</span><span class="p">(</span><span class="n">heter_rate</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="s">&quot;F&quot;</span><span class="p">)</span>
<span class="n">std</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="nf">sd</span><span class="p">(</span><span class="n">sample_missing_rate_het</span><span class="o">$</span><span class="n">heter_rate</span><span class="p">)</span><span class="w"> </span><span class="c1">## calculate standard deviation</span>
<span class="n">std_list</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="nf">c</span><span class="p">(</span><span class="n">std</span><span class="p">,</span><span class="w"> </span><span class="o">-</span><span class="n">std</span><span class="p">,</span><span class="w"> </span><span class="m">2</span><span class="o">*</span><span class="n">std</span><span class="p">,</span><span class="w"> </span><span class="m">-2</span><span class="o">*</span><span class="n">std</span><span class="p">,</span><span class="w"> </span><span class="m">3</span><span class="o">*</span><span class="n">std</span><span class="p">,</span><span class="w"> </span><span class="m">-3</span><span class="o">*</span><span class="n">std</span><span class="p">)</span>

<span class="nf">ggplot</span><span class="p">(</span><span class="n">sample_missing_rate_het</span><span class="p">,</span><span class="w"> </span><span class="nf">aes</span><span class="p">(</span><span class="n">x</span><span class="o">=</span><span class="w"> </span><span class="n">F_MISS</span><span class="p">,</span><span class="w"> </span><span class="n">y</span><span class="o">=</span><span class="w"> </span><span class="n">heter_rate</span><span class="p">))</span><span class="w"> </span><span class="o">+</span>
<span class="w">        </span><span class="nf">geom_point</span><span class="p">()</span><span class="w"> </span><span class="o">+</span>
<span class="w">        </span><span class="nf">geom_vline</span><span class="p">(</span><span class="n">xintercept</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="m">0.1</span><span class="p">,</span><span class="w"> </span><span class="n">linetype</span><span class="o">=</span><span class="s">&quot;dashed&quot;</span><span class="p">,</span><span class="w"> </span><span class="n">color</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="s">&quot;blue&quot;</span><span class="p">)</span><span class="w"> </span><span class="o">+</span>
<span class="w">        </span><span class="nf">geom_hline</span><span class="p">(</span><span class="n">yintercept</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="n">std_list</span><span class="p">[</span><span class="m">1</span><span class="o">:</span><span class="m">4</span><span class="p">],</span><span class="w"> </span><span class="n">col</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="s">&quot;grey&quot;</span><span class="p">,</span><span class="w"> </span><span class="n">linetype</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="s">&quot;dashed&quot;</span><span class="p">)</span><span class="w"> </span><span class="o">+</span>
<span class="w">        </span><span class="nf">geom_hline</span><span class="p">(</span><span class="n">yintercept</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="n">std_list</span><span class="p">[</span><span class="m">5</span><span class="o">:</span><span class="m">6</span><span class="p">],</span><span class="w"> </span><span class="n">col</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="s">&quot;red&quot;</span><span class="p">,</span><span class="w"> </span><span class="n">linetype</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="s">&quot;dashed&quot;</span><span class="p">)</span><span class="w"> </span><span class="o">+</span>
<span class="w">        </span><span class="nf">ggtitle</span><span class="p">(</span><span class="s">&quot;Heterozygosity vs Missingness across samples&quot;</span><span class="p">)</span><span class="w"> </span><span class="o">+</span>
<span class="w">        </span><span class="nf">theme_bw</span><span class="p">()</span><span class="w"> </span><span class="o">+</span>
<span class="w">        </span><span class="nf">xlab</span><span class="p">(</span><span class="s">&quot;Proportion of missing SNPs&quot;</span><span class="p">)</span><span class="w"> </span><span class="o">+</span>
<span class="w">        </span><span class="nf">ylab</span><span class="p">(</span><span class="s">&quot;Heterozygosity rate(+- 3 sd)&quot;</span><span class="p">)</span>
</pre></div>
</div>
<div class="admonition note">
<p class="admonition-title">Note</p>
<dl class="simple">
<dt><strong>Advice</strong>      Exclude samples with more than 3 standard deviations away from mean heterozygosity.</dt><dd><p>Lifelines uses 4sd from the mean heterozygosity.</p>
</dd>
</dl>
</div>
</section>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="Check_MAF.html" class="btn btn-neutral float-left" title="2. Check MAF (SNP-level)" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
        <a href="Check_sex_discordance.html" class="btn btn-neutral float-right" title="4. Check sex discordance (Sample-level)" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2023, Martin Banchero, Tatiana Karp.</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>