<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8" /><meta name="generator" content="Docutils 0.18.1: http://docutils.sourceforge.net/" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>SNP/GWAS workflow documentation! &mdash; SNP/GWAS workflow 2023 documentation</title>
      <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
      <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
      <link rel="stylesheet" href="_static/css/custom.css" type="text/css" />
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
        <script src="_static/jquery.js?v=5d32c60e"></script>
        <script src="_static/_sphinx_javascript_frameworks_compat.js?v=2cd50e6c"></script>
        <script src="_static/documentation_options.js?v=a4664895"></script>
        <script src="_static/doctools.js?v=888ff710"></script>
        <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="_static/js/theme.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Set-up" href="setup.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="#">
            
          </a>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <p class="caption" role="heading"><span class="caption-text">Introduction</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="setup.html">Set-up</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">From raw data to plink</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="genomestudio.html">From raw to plink</a></li>
<li class="toctree-l1"><a class="reference internal" href="genomestudio.html#load-data">Load data</a></li>
<li class="toctree-l1"><a class="reference internal" href="genomestudio.html#re-cluster">Re-cluster</a></li>
<li class="toctree-l1"><a class="reference internal" href="genomestudio.html#genomestudio-output-files">genomestudio output files</a></li>
<li class="toctree-l1"><a class="reference internal" href="genomestudio.html#authors">Authors</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Quality control</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="qualitycontrol.html">1. Check call rate per SNP and per sample (both SNP and Sample-level)</a></li>
<li class="toctree-l1"><a class="reference internal" href="qualitycontrol.html#check-maf-snp-level">2. Check MAF (SNP-level)</a></li>
<li class="toctree-l1"><a class="reference internal" href="qualitycontrol.html#check-sample-heterozygosity-sample-level">3. Check sample heterozygosity (Sample-level)</a></li>
<li class="toctree-l1"><a class="reference internal" href="qualitycontrol.html#check-sex-discordance-sample-level">4. Check sex discordance (Sample-level)</a></li>
<li class="toctree-l1"><a class="reference internal" href="qualitycontrol.html#hardy-weinberg-equilibrium-snp-level">6. Hardy–Weinberg equilibrium (SNP level)</a></li>
<li class="toctree-l1"><a class="reference internal" href="qualitycontrol.html#sample-qc-pca">7. Sample QC: PCA</a></li>
<li class="toctree-l1"><a class="reference internal" href="qualitycontrol.html#final-remarks">Final remarks</a></li>
<li class="toctree-l1"><a class="reference internal" href="qualitycontrol.html#author">Author</a></li>
<li class="toctree-l1"><a class="reference internal" href="qualitycontrol.html#references">References</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">GWAS</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="gwide.html">GWAS software</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="#">SNP/GWAS workflow</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="#" class="icon icon-home" aria-label="Home"></a></li>
      <li class="breadcrumb-item active">SNP/GWAS workflow documentation!</li>
      <li class="wy-breadcrumbs-aside">
              <a href="https://github.com///blob//docs/index.md" class="fa fa-github"> Edit on GitHub</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section class="tex2jax_ignore mathjax_ignore" id="snp-gwas-workflow-documentation">
<h1>SNP/GWAS workflow documentation!<a class="headerlink" href="#snp-gwas-workflow-documentation" title="Link to this heading"></a></h1>
<section id="snp-array-quality-control-protocol">
<h2>SNP array quality control protocol<a class="headerlink" href="#snp-array-quality-control-protocol" title="Link to this heading"></a></h2>
<p>This file describes step to perform quality control for the SNP array data.
You will need genotype data in plink format (either .ped and .map or .bed file)</p>
<p>The .ped file contains information on each study participant including family ID,
participant ID, father ID, mother ID, sex, phenotype, and the full typed genotype. Here, each SNP
is bi-allelic (i.e., only two nucleotides are observed at any given SNP across study participants) and
coded as a pair of nucleotides (A, C, T, or G). Notably, the ordering in the pair is non-informative
in the sense that the first alleles listed for each of the two SNPs are not necessarily on the same
chromosome. The .map file contains a row for each SNP corresponding
chromosome (chr) and coordinate (BPPos) based on the current genome build.</p>
<p>Most of the steps for the QC will be performed in plink1.9, so it has to be installed.</p>
<p>Let’s prepare the environment by uploading the libraries</p>
<div class="toctree-wrapper compound">
</div>
<div class="toctree-wrapper compound">
</div>
<div class="toctree-wrapper compound">
</div>
<div class="toctree-wrapper compound">
</div>
</section>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="setup.html" class="btn btn-neutral float-right" title="Set-up" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2023, MIT.</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>