Search.setIndex({"docnames": ["Check_MAF", "Check_call_rate_per_SNP_per_sample", "Check_relatedness", "Check_sample_heterozygosity", "Check_sex_discordance", "GWAS", "Hardy--Weinberg_equilibrium", "Processing_raw_data", "QC_overview", "index"], "filenames": ["Check_MAF.rst", "Check_call_rate_per_SNP_per_sample.rst", "Check_relatedness.rst", "Check_sample_heterozygosity.rst", "Check_sex_discordance.rst", "GWAS.rst", "Hardy--Weinberg_equilibrium.rst", "Processing_raw_data.rst", "QC_overview.rst", "index.rst"], "titles": ["2. Check MAF (SNP-level)", "1. Check call rate per SNP and per sample (both SNP and Sample-level)", "5. Check relatedness (Sample-level)", "3. Check sample heterozygosity (Sample-level)", "4. Check sex discordance (Sample-level)", "GWAS", "6. Hardy\u2013Weinberg equilibrium (SNP level)", "Raw data processing", "<strong>SNP array quality control protocol</strong>", "Welcome to genotype data analysis UMCG-GRIAC\u2019s documentation!"], "terms": {"index": [], "modul": [], "search": [], "page": [], "A": [0, 2, 8], "flexibl": [], "r": 1, "packag": 8, "process": [3, 9], "data": [1, 2, 8], "from": [2, 3, 7, 8, 9], "differ": [], "plate": [], "reader": [], "machin": [], "The": [1, 8, 9], "main": [], "function": 8, "i": [0, 1, 2, 4, 5, 7, 8, 9], "provid": 6, "simpl": [], "method": 4, "can": [0, 2, 6], "raw": [], "micropl": [], "like": [], "spectramax": [], "fluorstar": [], "multiscango": [], "appli": 2, "thi": [0, 2, 4, 5, 7, 8, 9], "gener": [0, 2], "tidi": [], "tibbl": [], "object": [], "combin": [], "layout": [], "file": [1, 4, 6, 8], "growth": [], "rate": [2, 4], "ar": [2, 4, 8], "calcul": [], "plot": [], "over": [], "frame": 1, "critic": [], "point": [], "sever": [], "research": [], "line": [], "perform": [2, 8], "experi": [], "us": [2, 3, 4, 7], "mainli": [], "due": 3, "time": [], "take": [], "clean": [], "sort": [], "format": [1, 8], "also": [], "featur": [], "allow": [], "detect": 4, "which": 1, "well": [], "present": [], "faulti": [], "e": [2, 8], "g": [2, 8], "fill": [], "correctli": [], "anoth": [], "veri": 0, "toctre": [], "section": [], "api": [], "genotyp": [2, 3, 4, 7, 8], "da": 9, "workflow": [5, 7, 9], "cover": [5, 7, 9], "intens": [7, 9], "imput": 9, "gwa": [2, 9], "analysi": [2, 5], "note": [], "project": [5, 7, 9], "under": [0, 5, 7, 9], "activ": [5, 7, 9], "develop": [5, 7, 9], "plink": 8, "step": [2, 4, 8], "qc": 8, "overview": [], "qualiti": [], "control": 6, "genom": 8, "wide": [], "associ": [], "studi": [0, 1, 2, 8], "read": [1, 7], "intes": 7, "hand": [], "studio": [], "minor": 0, "allel": [0, 3, 8], "frequenc": 0, "larg": [0, 2], "degre": 0, "homogen": 0, "given": [0, 1, 3, 8], "across": [0, 2, 8], "particip": [0, 8], "result": 0, "inadequ": 0, "power": 0, "infer": 0, "statist": [0, 6], "signific": 0, "relationship": 0, "between": [0, 2], "trait": 0, "occur": 0, "when": [0, 4], "we": [0, 2, 3, 4], "have": [0, 6], "small": 0, "so": [0, 8], "major": 0, "individu": [0, 1, 2, 3], "two": [0, 3, 8], "copi": 0, "threshold": [0, 2, 6], "vari": [0, 2, 6], "1": 0, "5": [0, 6], "sampl": 0, "set": 0, "advic": 0, "For": [0, 4], "size": 0, "recommend": 0, "exclud": [0, 3, 4], "0": [0, 2], "05": 0, "defin": 1, "proport": 1, "correspond": [1, 8], "inform": [1, 4, 8], "miss": 1, "imiss": 1, "lmiss": 1, "runplink": [1, 8], "ped": [1, 6, 8], "map": [1, 8], "out": 1, "file_call_r": 1, "chang": [1, 8], "system": [1, 8], "cat": 1, "tr": 1, "": [1, 8], "t": [1, 6, 8], "call_rate_snp": 1, "tab": 1, "load": 1, "delim": 1, "select": 1, "x": [1, 4], "call_rate_sampl": 1, "popul": 2, "base": [2, 8], "cohort": 2, "often": 2, "limit": 2, "unrel": 2, "linear": 2, "model": 2, "approach": 2, "assum": 2, "independ": 2, "importantli": 2, "region": 2, "hospit": 2, "complex": 2, "diseas": 2, "same": [2, 4, 8], "famili": [2, 8], "recruit": 2, "unintention": 2, "common": 2, "measur": 2, "duplic": 2, "pair": [2, 8], "an": [2, 3], "kinship": 2, "coeffici": 2, "greater": 2, "than": [2, 3], "1875": 2, "mai": [2, 3], "suggest": 2, "mixtur": 2, "typic": 2, "relat": 2, "lower": 2, "call": [2, 4], "remov": 2, "first": [2, 8], "need": [2, 4, 8], "linkag": 2, "disequilibrium": 2, "valu": [2, 6], "7": 2, "could": 2, "some": 2, "2": 2, "snp": [2, 3, 4], "high": [2, 3], "each": [2, 3, 4, 8], "other": 2, "one": 2, "elimin": 2, "redund": 2, "dimens": 2, "reduct": 2, "commonli": 2, "prior": 2, "both": 2, "pca": 2, "refer": 3, "presenc": 3, "within": 3, "outlier": 3, "show": 3, "excess": 3, "deplet": 3, "heterozygot": 3, "contamin": 3, "issu": 3, "dure": 3, "should": 3, "do": 3, "alreadi": [3, 4], "onli": [3, 8], "advis": 3, "more": 3, "standard": 3, "deviat": 3, "awai": 3, "mean": 3, "lifelin": 3, "4sd": 3, "mix": 4, "up": 4, "investig": 4, "look": 4, "mismatch": 4, "where": [4, 8], "compar": 4, "chromosom": [4, 8], "y": 4, "nb": 4, "howev": 4, "doe": [4, 6], "reliabl": 4, "abnorm": 4, "low": 4, "check": 6, "phenotyp": [6, 8], "If": 6, "doesn": 6, "reformat": 6, "add": 6, "code": [6, 8], "hwe": 6, "test": 6, "consid": 6, "p": 6, "10e": 6, "describ": [8, 9], "you": 8, "either": 8, "bed": 8, "contain": 8, "includ": 8, "id": 8, "father": 8, "mother": 8, "sex": 8, "full": 8, "type": 8, "here": 8, "bi": 8, "nucleotid": 8, "observ": 8, "ani": 8, "c": 8, "notabl": 8, "order": 8, "non": 8, "sens": 8, "list": 8, "necessarili": 8, "row": 8, "chr": 8, "coordin": 8, "bppo": 8, "current": 8, "build": 8, "most": 8, "plink1": 8, "9": 8, "ha": 8, "instal": 8, "let": 8, "prepar": 8, "environ": 8, "upload": 8, "librari": 8, "creat": 8, "run": 8, "directori": 8, "ex": 8, "plinkopt": 8, "past": 8, "might": 8, "depend": 8, "wai": 8, "plan": 8, "rational": 9, "follow": 9, "figur": 9, "make": 1, "easier": 1}, "objects": {}, "objtypes": {}, "objnames": {}, "titleterms": {"welcom": 9, "genotypeda": [], "umcg": 9, "griac": 9, "": 9, "document": 9, "indic": [], "tabl": [], "mpxtractor": [], "content": [], "gwa": 5, "raw": 7, "data": [7, 9], "process": 7, "qualiti": 8, "control": 8, "workflow": [], "hand": 7, "genom": 7, "studio": 7, "gener": 7, "plink": 7, "file": 7, "2": 0, "check": [0, 1, 2, 3, 4], "maf": 0, "snp": [0, 1, 6, 8], "level": [0, 1, 2, 3, 4, 6], "plot": [0, 1, 3], "histogram": [0, 1], "1": 1, "call": [1, 3], "rate": [1, 3], "per": 1, "sampl": [1, 2, 3, 4], "both": 1, "5": 2, "related": 2, "ld": 2, "prune": 2, "calcul": 2, "ibd": 2, "ident": 2, "descent": 2, "degre": 2, "recent": 2, "share": 2, "ancestri": 2, "3": 3, "heterozygos": 3, "v": 3, "4": 4, "sex": 4, "discord": 4, "6": 6, "hardi": 6, "weinberg": 6, "equilibrium": 6, "arrai": 8, "protocol": 8, "introduct": [], "genotyp": 9, "analysi": 9}, "envversion": {"sphinx.domains.c": 3, "sphinx.domains.changeset": 1, "sphinx.domains.citation": 1, "sphinx.domains.cpp": 9, "sphinx.domains.index": 1, "sphinx.domains.javascript": 3, "sphinx.domains.math": 2, "sphinx.domains.python": 4, "sphinx.domains.rst": 2, "sphinx.domains.std": 2, "sphinx": 60}, "alltitles": {"2. Check MAF (SNP-level)": [[0, "check-maf-snp-level"]], "Plot MAF histogram": [[0, "plot-maf-histogram"]], "1. Check call rate per SNP and per sample (both SNP and Sample-level)": [[1, "check-call-rate-per-snp-and-per-sample-both-snp-and-sample-level"]], "Plot histograms of call rate per sample": [[1, "plot-histograms-of-call-rate-per-sample"]], "Plot histograms of call rate per snp": [[1, "plot-histograms-of-call-rate-per-snp"]], "5. Check relatedness (Sample-level)": [[2, "check-relatedness-sample-level"]], "LD pruning": [[2, "ld-pruning"]], "Calculate IBD - identity by descent (the degree of recent shared ancestry)": [[2, "calculate-ibd-identity-by-descent-the-degree-of-recent-shared-ancestry"]], "3. Check sample heterozygosity (Sample-level)": [[3, "check-sample-heterozygosity-sample-level"]], "Plot sample call rate vs heterozygosity": [[3, "plot-sample-call-rate-vs-heterozygosity"]], "4. Check sex discordance (Sample-level)": [[4, "check-sex-discordance-sample-level"]], "GWAS": [[5, "gwas"]], "6. Hardy\u2013Weinberg equilibrium (SNP level)": [[6, "hardy-weinberg-equilibrium-snp-level"]], "Raw data processing": [[7, "raw-data-processing"]], "Hands on genome studio": [[7, "hands-on-genome-studio"]], "Generate plink files": [[7, "generate-plink-files"]], "SNP array quality control protocol": [[8, "snp-array-quality-control-protocol"]], "Welcome to genotype data analysis UMCG-GRIAC\u2019s documentation!": [[9, "welcome-to-genotype-data-analysis-umcg-griac-s-documentation"]]}, "indexentries": {}})