<<<<<<< HEAD
Search.setIndex({"docnames": ["Check_MAF", "Check_call_rate_per_SNP_per_sample", "Check_relatedness", "Check_sample_heterozygosity", "Check_sex_discordance", "GWAS", "Hardy--Weinberg_equilibrium", "Processing_raw_data", "QC_overview", "index"], "filenames": ["Check_MAF.rst", "Check_call_rate_per_SNP_per_sample.rst", "Check_relatedness.rst", "Check_sample_heterozygosity.rst", "Check_sex_discordance.rst", "GWAS.rst", "Hardy--Weinberg_equilibrium.rst", "Processing_raw_data.rst", "QC_overview.rst", "index.rst"], "titles": ["2. Check MAF (SNP-level)", "1. Check call rate per SNP and per sample (both SNP and Sample-level)", "5. Check relatedness (Sample-level)", "3. Check sample heterozygosity (Sample-level)", "4. Check sex discordance (Sample-level)", "GWAS", "6. Hardy\u2013Weinberg equilibrium (SNP level)", "Raw data processing", "<strong>SNP array quality control protocol</strong>", "Welcome to GenotypeDA-UMCG-GRIAC\u2019s documentation!"], "terms": {"index": [], "modul": [], "search": [], "page": [], "A": [0, 2, 8], "flexibl": [], "r": 1, "packag": 8, "process": [3, 9], "data": [1, 2, 8, 9], "from": [2, 3, 7, 8, 9], "differ": [], "plate": [], "reader": [], "machin": [], "The": [1, 8, 9], "main": [], "function": 8, "i": [0, 1, 2, 4, 5, 7, 8, 9], "provid": 6, "simpl": [], "method": 4, "can": [0, 2, 6], "raw": [], "micropl": [], "like": [], "spectramax": [], "fluorstar": [], "multiscango": [], "appli": 2, "thi": [0, 2, 4, 5, 7, 8, 9], "gener": [0, 2], "tidi": [], "tibbl": [], "object": [], "combin": [], "layout": [], "file": [1, 4, 6, 8], "growth": [], "rate": [2, 4], "ar": [2, 4, 8], "calcul": [], "plot": [], "over": [], "frame": 1, "critic": [], "point": [], "sever": [], "research": [], "line": [], "perform": [2, 8], "experi": [], "us": [2, 3, 4, 7], "mainli": [], "due": 3, "time": [], "take": [], "clean": [], "sort": [], "format": [1, 8], "also": [], "featur": [], "allow": [], "detect": 4, "which": 1, "well": [], "present": [], "faulti": [], "e": [2, 8], "g": [2, 8], "fill": [], "correctli": [], "anoth": [], "veri": 0, "toctre": [], "section": [], "api": [], "genotyp": [2, 3, 4, 7, 8, 9], "da": 9, "workflow": [5, 7, 9], "cover": [5, 7, 9], "intens": [7, 9], "imput": 9, "gwa": [2, 9], "analysi": [2, 5, 9], "note": [], "project": [5, 7, 9], "under": [0, 5, 7, 9], "activ": [5, 7, 9], "develop": [5, 7, 9], "plink": 8, "step": [2, 4, 8], "qc": 8, "overview": [], "qualiti": [], "control": 6, "genom": 8, "wide": [], "associ": [], "studi": [0, 1, 2, 8], "read": [1, 7], "intes": 7, "hand": [], "studio": [], "minor": 0, "allel": [0, 3, 8], "frequenc": 0, "larg": [0, 2], "degre": 0, "homogen": 0, "given": [0, 1, 3, 8], "across": [0, 2, 8], "particip": [0, 8], "result": 0, "inadequ": 0, "power": 0, "infer": 0, "statist": [0, 6], "signific": 0, "relationship": 0, "between": [0, 2], "trait": 0, "occur": 0, "when": [0, 4], "we": [0, 2, 3, 4], "have": [0, 6], "small": 0, "so": [0, 8], "major": 0, "individu": [0, 1, 2, 3], "two": [0, 3, 8], "copi": 0, "threshold": [0, 2, 6], "vari": [0, 2, 6], "1": 0, "5": [0, 6], "sampl": 0, "set": 0, "advic": 0, "For": [0, 4], "size": 0, "recommend": 0, "exclud": [0, 3, 4], "0": [0, 2], "05": 0, "defin": 1, "proport": 1, "correspond": [1, 8], "inform": [1, 4, 8], "miss": 1, "imiss": 1, "lmiss": 1, "runplink": [1, 8], "ped": [1, 6, 8], "map": [1, 8], "out": 1, "file_call_r": 1, "chang": [1, 8], "system": [1, 8], "cat": 1, "tr": 1, "": [1, 8], "t": [1, 6, 8], "call_rate_snp": 1, "tab": 1, "load": 1, "delim": 1, "select": 1, "x": [1, 4], "call_rate_sampl": 1, "popul": 2, "base": [2, 8], "cohort": 2, "often": 2, "limit": 2, "unrel": 2, "linear": 2, "model": 2, "approach": 2, "assum": 2, "independ": 2, "importantli": 2, "region": 2, "hospit": 2, "complex": 2, "diseas": 2, "same": [2, 4, 8], "famili": [2, 8], "recruit": 2, "unintention": 2, "common": 2, "measur": 2, "duplic": 2, "pair": [2, 8], "an": [2, 3], "kinship": 2, "coeffici": 2, "greater": 2, "than": [2, 3], "1875": 2, "mai": [2, 3], "suggest": 2, "mixtur": 2, "typic": 2, "relat": 2, "lower": 2, "call": [2, 4], "remov": 2, "first": [2, 8], "need": [2, 4, 8], "linkag": 2, "disequilibrium": 2, "valu": [2, 6], "7": 2, "could": 2, "some": 2, "2": 2, "snp": [2, 3, 4], "high": [2, 3], "each": [2, 3, 4, 8], "other": 2, "one": 2, "elimin": 2, "redund": 2, "dimens": 2, "reduct": 2, "commonli": 2, "prior": 2, "both": 2, "pca": 2, "refer": 3, "presenc": 3, "within": 3, "outlier": 3, "show": 3, "excess": 3, "deplet": 3, "heterozygot": 3, "contamin": 3, "issu": 3, "dure": 3, "should": 3, "do": 3, "alreadi": [3, 4], "onli": [3, 8], "advis": 3, "more": 3, "standard": 3, "deviat": 3, "awai": 3, "mean": 3, "lifelin": 3, "4sd": 3, "mix": 4, "up": 4, "investig": 4, "look": 4, "mismatch": 4, "where": [4, 8], "compar": 4, "chromosom": [4, 8], "y": 4, "nb": 4, "howev": 4, "doe": [4, 6], "reliabl": 4, "abnorm": 4, "low": 4, "check": 6, "phenotyp": [6, 8], "If": 6, "doesn": 6, "reformat": 6, "add": 6, "code": [6, 8], "hwe": 6, "test": 6, "consid": 6, "p": 6, "10e": 6, "describ": [8, 9], "you": 8, "either": 8, "bed": 8, "contain": 8, "includ": 8, "id": 8, "father": 8, "mother": 8, "sex": 8, "full": 8, "type": 8, "here": 8, "bi": 8, "nucleotid": 8, "observ": 8, "ani": 8, "c": 8, "notabl": 8, "order": 8, "non": 8, "sens": 8, "list": 8, "necessarili": 8, "row": 8, "chr": 8, "coordin": 8, "bppo": 8, "current": 8, "build": 8, "most": 8, "plink1": 8, "9": 8, "ha": 8, "instal": 8, "let": 8, "prepar": 8, "environ": 8, "upload": 8, "librari": 8, "creat": 8, "run": 8, "directori": 8, "ex": 8, "plinkopt": 8, "past": 8, "might": 8, "depend": 8, "wai": 8, "plan": 8, "rational": 9, "follow": 9, "figur": 9, "make": 1, "easier": 1}, "objects": {}, "objtypes": {}, "objnames": {}, "titleterms": {"welcom": 9, "genotypeda": 9, "umcg": 9, "griac": 9, "": 9, "document": 9, "indic": [], "tabl": [], "mpxtractor": [], "content": [], "gwa": 5, "raw": 7, "data": 7, "process": 7, "qualiti": 8, "control": 8, "workflow": [], "hand": 7, "genom": 7, "studio": 7, "gener": 7, "plink": 7, "file": 7, "2": 0, "check": [0, 1, 2, 3, 4], "maf": 0, "snp": [0, 1, 6, 8], "level": [0, 1, 2, 3, 4, 6], "plot": [0, 1, 3], "histogram": [0, 1], "1": 1, "call": [1, 3], "rate": [1, 3], "per": 1, "sampl": [1, 2, 3, 4], "both": 1, "5": 2, "related": 2, "ld": 2, "prune": 2, "calcul": 2, "ibd": 2, "ident": 2, "descent": 2, "degre": 2, "recent": 2, "share": 2, "ancestri": 2, "3": 3, "heterozygos": 3, "v": 3, "4": 4, "sex": 4, "discord": 4, "6": 6, "hardi": 6, "weinberg": 6, "equilibrium": 6, "arrai": 8, "protocol": 8}, "envversion": {"sphinx.domains.c": 3, "sphinx.domains.changeset": 1, "sphinx.domains.citation": 1, "sphinx.domains.cpp": 9, "sphinx.domains.index": 1, "sphinx.domains.javascript": 3, "sphinx.domains.math": 2, "sphinx.domains.python": 4, "sphinx.domains.rst": 2, "sphinx.domains.std": 2, "sphinx": 60}, "alltitles": {"GWAS": [[5, "gwas"]], "Raw data processing": [[7, "raw-data-processing"]], "Hands on genome studio": [[7, "hands-on-genome-studio"]], "Generate plink files": [[7, "generate-plink-files"]], "2. Check MAF (SNP-level)": [[0, "check-maf-snp-level"]], "Plot MAF histogram": [[0, "plot-maf-histogram"]], "5. Check relatedness (Sample-level)": [[2, "check-relatedness-sample-level"]], "LD pruning": [[2, "ld-pruning"]], "Calculate IBD - identity by descent (the degree of recent shared ancestry)": [[2, "calculate-ibd-identity-by-descent-the-degree-of-recent-shared-ancestry"]], "3. Check sample heterozygosity (Sample-level)": [[3, "check-sample-heterozygosity-sample-level"]], "Plot sample call rate vs heterozygosity": [[3, "plot-sample-call-rate-vs-heterozygosity"]], "4. Check sex discordance (Sample-level)": [[4, "check-sex-discordance-sample-level"]], "6. Hardy\u2013Weinberg equilibrium (SNP level)": [[6, "hardy-weinberg-equilibrium-snp-level"]], "SNP array quality control protocol": [[8, "snp-array-quality-control-protocol"]], "Welcome to GenotypeDA-UMCG-GRIAC\u2019s documentation!": [[9, "welcome-to-genotypeda-umcg-griac-s-documentation"]], "1. Check call rate per SNP and per sample (both SNP and Sample-level)": [[1, "check-call-rate-per-snp-and-per-sample-both-snp-and-sample-level"]], "Plot histograms of call rate per sample": [[1, "plot-histograms-of-call-rate-per-sample"]], "Plot histograms of call rate per snp": [[1, "plot-histograms-of-call-rate-per-snp"]]}, "indexentries": {}})
=======
Search.setIndex({"docnames": ["genomestudio", "gwide", "index", "qualitycontrol", "setup"], "filenames": ["genomestudio.md", "gwide.md", "index.md", "qualitycontrol.md", "setup.md"], "titles": ["Load data", "GWAS software", "SNP/GWAS workflow documentation!", "1. Check call rate per SNP and per sample (both SNP and Sample-level)", "Set-up"], "terms": {"index": [], "modul": [], "search": [], "page": [], "A": [2, 3], "flexibl": [], "r": [3, 4], "packag": 4, "process": 3, "data": [2, 3], "from": [3, 4], "differ": 3, "plate": [], "reader": [], "machin": [], "The": [2, 3], "main": [], "function": [3, 4], "i": [2, 3, 4], "provid": 3, "simpl": [], "method": 3, "can": 3, "raw": [], "micropl": [], "like": [], "spectramax": [], "fluorstar": [], "multiscango": [], "appli": 3, "thi": [0, 2, 3, 4], "gener": [0, 3], "tidi": [], "tibbl": [], "object": [], "combin": [], "layout": [], "file": [2, 3, 4], "growth": [], "rate": [], "ar": 2, "calcul": [], "plot": [], "over": [], "frame": 3, "critic": [], "point": 0, "sever": [], "research": 3, "line": [], "perform": [2, 3, 4], "experi": [], "us": 3, "mainli": [], "due": 3, "time": [], "take": 3, "clean": [], "sort": [], "format": [2, 3], "also": [], "featur": [], "allow": [], "detect": 3, "which": 3, "well": [], "present": [], "faulti": [], "e": [2, 3], "g": [2, 3], "fill": [], "correctli": [], "anoth": [], "veri": 3, "toctre": [], "section": [], "api": [], "genotyp": [2, 3], "da": [], "workflow": [], "cover": [], "intens": [], "imput": 3, "gwa": 3, "analysi": 3, "note": [], "project": [], "under": [1, 3], "activ": [], "develop": 1, "plink": [2, 3, 4], "step": [2, 4], "qc": [0, 4], "overview": [], "qualiti": 3, "control": 3, "genom": [2, 3], "wide": 3, "associ": 3, "studi": [2, 3], "read": 3, "intes": [], "hand": [], "studio": [], "Will": 4, "need": [2, 3, 4], "gnomestudio": 4, "plink1": 4, "9": [3, 4], "we": [0, 3], "list": [2, 3], "go": [], "defin": 3, "insid": [], "runplink": [3, 4], "plinkopt": 4, "system": [3, 4], "past": 4, "most": 4, "so": [3, 4], "ha": 4, "instal": 4, "let": [3, 4], "": [3, 4], "prepar": 4, "environ": 4, "upload": 4, "librari": 4, "creat": [3, 4], "run": 4, "directori": 4, "where": [3, 4], "ex": 4, "you": [2, 3, 4], "might": [3, 4], "chang": [3, 4], "depend": [3, 4], "wai": 4, "plan": [3, 4], "dplyr": 4, "ggplot2": 4, "ggrepel": 4, "describ": 2, "either": 2, "ped": [0, 2, 3], "map": [0, 2, 3], "bed": [2, 3], "contain": 2, "inform": [2, 3], "each": [2, 3], "particip": [2, 3], "includ": 2, "famili": [2, 3], "id": 2, "father": 2, "mother": 2, "sex": 2, "phenotyp": [2, 3], "full": 2, "type": 2, "here": 2, "bi": 2, "allel": [2, 3], "onli": [2, 3], "two": [2, 3], "nucleotid": 2, "observ": [2, 3], "ani": 2, "given": [2, 3], "across": [2, 3], "code": [2, 3], "pair": [2, 3], "c": [2, 3], "t": [2, 3], "notabl": 2, "order": 2, "non": 2, "sens": 2, "first": [2, 3], "necessarili": 2, "same": [2, 3], "chromosom": 2, "row": 2, "correspond": [2, 3], "chr": 2, "coordin": 2, "bppo": 2, "base": [2, 3], "current": [2, 3], "build": [2, 3], "proport": 3, "individu": 3, "miss": 3, "out": 3, "file_call_r": 3, "make": 3, "easier": 3, "cat": 3, "tr": 3, "call_rate_sampl": 3, "tab": 3, "call_rate_snp": 3, "histogram": [], "load": 3, "delim": 3, "select": [0, 3], "x": 0, "n": 3, "filter": [], "our": 3, "0": 3, "n_filtered_out": 3, "sum": 3, "f_miss": 3, "ggplot": 3, "ae": 3, "100": 3, "geom_histogram": 3, "bin": 3, "50": 3, "theme_bw": 3, "ylab": 3, "number": 3, "xlab": 3, "geom_vlin": 3, "xintercept": 3, "90": 3, "color": 3, "red": 3, "linetyp": 3, "dash": 3, "annot": 3, "geom": 3, "text": 3, "70": 3, "y": 0, "100000": 3, "label": 3, "paste0": 3, "exclud": 3, "size": 3, "4": [], "again": 3, "geno": 3, "recod": 3, "file_filtered_snp": 3, "file_call_rate_filtered_snp": 3, "file_call_rate_filtered_snps_imiss": 3, "call_rate_samples_after_filt": 3, "after": [], "iter": 3, "help": 3, "decid": 3, "threshold": 3, "choos": 3, "low": 3, "usual": 3, "vari": 3, "95": 3, "98": 3, "200": 3, "minor": 3, "frequenc": 3, "larg": 3, "degre": 3, "homogen": 3, "result": 3, "inadequ": 3, "power": 3, "infer": 3, "statist": 3, "signific": 3, "relationship": 3, "between": 3, "trait": 3, "occur": 3, "when": 3, "have": 3, "small": 3, "major": 3, "copi": 3, "5": [], "set": 3, "freq": 3, "file_maf": 3, "frq": 3, "file_maf_frq": 3, "maf_check": 3, "maf_01": 3, "01": 3, "maf_05": 3, "05": 3, "grei": 3, "02": 3, "angl": 3, "3": [], "043": 3, "For": 3, "recommend": 3, "refer": [], "presenc": 3, "within": 3, "an": 3, "outlier": 3, "show": 3, "excess": 3, "deplet": 3, "heterozygot": 3, "mai": 3, "contamin": 3, "issu": 3, "dure": 3, "should": [0, 3], "do": 3, "alreadi": 3, "high": 3, "het": 3, "file_het": 3, "het_level": 3, "more": 3, "than": 3, "standard": 3, "deviat": 3, "awai": 3, "mean": 3, "lifelin": 3, "4sd": 3, "sample_missing_rate_het": 3, "left_join": 3, "iid": 3, "renam": 3, "heter_r": 3, "f": [], "std": 3, "sd": 3, "std_list": 3, "geom_point": 3, "blue": 3, "geom_hlin": 3, "yintercept": 3, "col": 3, "6": [], "ggtitl": 3, "missing": 3, "mix": 3, "up": 3, "investig": 3, "look": 3, "mismatch": 3, "compar": 3, "nb": 3, "howev": 3, "doe": 3, "reliabl": 3, "abnorm": 3, "mind": 3, "file_snp_filt": 3, "now": 3, "sexcheck": 3, "file_snp_filtered_sexcheck": 3, "taken": 3, "plink_sex_filt": 3, "mutat": 3, "pedsex": 3, "if_els": 3, "femal": 3, "male": 3, "geom_jitt": 3, "ifels": 3, "geom_text_repel": 3, "hjust": 3, "vjust": 3, "mark": 3, "problemat": 3, "avail": 3, "advis": 3, "It": 3, "import": 3, "ass": 3, "visual": 3, "ycount": 3, "file_snp_filtered_ycount": 3, "file_snp_filtered_ycount_sexcheck": 3, "plink_sex_filtered_ycount": 3, "clearli": 3, "separ": 3, "axi": 3, "3000": 3, "count": [], "strang": [], "related": [], "popul": 3, "cohort": 3, "often": 3, "limit": 3, "unrel": 3, "linear": 3, "model": 3, "approach": 3, "assum": 3, "independ": 3, "importantli": 3, "region": 3, "hospit": 3, "complex": 3, "diseas": 3, "recruit": 3, "unintention": 3, "common": 3, "measur": 3, "duplic": 3, "ident": [], "descent": [], "ibd": [], "kinship": 3, "coeffici": 3, "greater": 3, "1875": 3, "suggest": 3, "mixtur": 3, "typic": 3, "relat": 3, "lower": 3, "remov": 3, "ld": [], "prune": [], "linkag": 3, "disequilibrium": 3, "could": 3, "some": 3, "other": 3, "one": 3, "elimin": 3, "redund": 3, "dimens": 3, "reduct": 3, "commonli": 3, "prior": 3, "indep": 3, "pairwis": 3, "keep": 3, "your": 3, "prunedset": 3, "recent": 3, "share": 3, "ancestri": 3, "z": 3, "pruned_ibd": 3, "zcat": 3, "gz": 3, "pi_hat": 3, "column": 3, "output": 3, "tabl": 3, "ibd_high_relat": 3, "print": 3, "violat": 3, "hwe": 3, "indic": 3, "substructur": 3, "occurr": 3, "error": 3, "while": 3, "thei": 3, "alwai": 3, "distinguish": 3, "practic": 3, "If": 3, "case": 3, "statu": 3, "departur": 3, "\ud835\udf122": 3, "good": 3, "fit": 3, "test": 3, "expect": 3, "doesn": 3, "reformat": 3, "add": 3, "consid": 3, "p": 3, "10e": 3, "only_control": 3, "controls_hardi": 3, "controls_hardy_hw": 3, "plink_controls_hw": 3, "unaff": 3, "identif": 3, "diverg": 3, "requir": 3, "dataset": 3, "divers": 3, "ethnic": 3, "hgdp": 3, "hapmap": 3, "exclus": 3, "question": 3, "all": 3, "filterd": 3, "delet": 3, "ind": 3, "fail": 3, "txt": 3, "qced": 3, "extra": 3, "swap": 3, "arrai": 3, "rna": 3, "seq": 3, "care": 3, "creation": 3, "michigan": 3, "server": 3, "vcf": 3, "bcftool": 3, "view": 3, "r2": 3, "oz": 3, "input_fil": 3, "output_fil": 3, "These": 3, "updat": 3, "document": 3, "turner": 3, "stephen": 3, "et": 3, "al": 3, "procedur": 3, "protocol": 3, "human": 3, "genet": 3, "68": 3, "2011": 3, "19": 3, "reed": 3, "eric": 3, "guid": 3, "post": 3, "analyt": 3, "interrog": 3, "medicin": 3, "34": 3, "28": 3, "2015": 3, "3769": 3, "3792": 3, "anderson": 3, "carl": 3, "natur": 3, "2010": 3, "1564": 3, "1573": 3, "To": [], "easi": [], "work": [], "imag": 0, "want": 0, "autosom": 0, "chromosoma": 0, "improv": 0, "At": 0, "abl": 0, "continu": 0, "martin": [0, 2], "banchero": [0, 2], "rui": [0, 2], "marcelo": [0, 2], "tatiana": [2, 3], "karp": [2, 3], "yidan": 2, "sun": 2}, "objects": {}, "objtypes": {}, "objnames": {}, "titleterms": {"welcom": [], "genotypeda": [], "umcg": [], "griac": [], "": [], "document": 2, "indic": [], "tabl": [], "mpxtractor": [], "content": [], "gwa": [1, 2], "raw": [], "data": 0, "process": [], "qualiti": 2, "control": 2, "workflow": 2, "hand": [], "genom": [], "studio": [], "gener": [], "plink": [], "file": 0, "snp": [2, 3], "set": 4, "up": 4, "softwar": [1, 4], "arrai": 2, "protocol": 2, "1": 3, "check": 3, "call": 3, "rate": 3, "per": 3, "sampl": 3, "both": 3, "level": 3, "imiss": 3, "lmiss": 3, "plot": 3, "histogram": 3, "exclud": [], "90": [], "again": [], "after": 3, "filter": 3, "2": 3, "maf": 3, "3": 3, "heterozygos": 3, "v": 3, "4": 3, "sex": 3, "discord": 3, "report": 3, "f": 3, "valu": 3, "x": 3, "y": 3, "chr": 3, "count": 3, "also": 3, "ar": 3, "strang": 3, "chromosom": 3, "6": 3, "hardi": 3, "weinberg": 3, "equilibrium": 3, "7": 3, "qc": 3, "pca": 3, "last": 3, "step": 3, "final": 3, "remark": 3, "refer": 3, "from": [], "load": 0, "re": 0, "cluster": 0, "genomestudio": 0, "output": 0, "author": [0, 2, 3], "acknowledg": 2, "5": 3, "related": 3, "ld": 3, "prune": 3, "calcul": 3, "ibd": 3, "ident": 3, "descent": 3}, "envversion": {"sphinx.domains.c": 3, "sphinx.domains.changeset": 1, "sphinx.domains.citation": 1, "sphinx.domains.cpp": 9, "sphinx.domains.index": 1, "sphinx.domains.javascript": 3, "sphinx.domains.math": 2, "sphinx.domains.python": 4, "sphinx.domains.rst": 2, "sphinx.domains.std": 2, "sphinx": 60}, "alltitles": {"Set-up": [[4, "set-up"]], "Software": [[4, "software"]], "GWAS software": [[1, "gwas-software"]], "Load data": [[0, "load-data"]], "Re-cluster": [[0, "re-cluster"]], "genomestudio output files": [[0, "genomestudio-output-files"]], "Authors": [[0, "authors"], [2, "authors"]], "SNP/GWAS workflow documentation!": [[2, "snp-gwas-workflow-documentation"]], "SNP array quality control protocol": [[2, "snp-array-quality-control-protocol"]], "Acknowledgements": [[2, "acknowledgements"]], "1. Check call rate per SNP and per sample (both SNP and Sample-level)": [[3, "check-call-rate-per-snp-and-per-sample-both-snp-and-sample-level"]], "Call rate per sample (imiss) and per SNP (lmiss)": [[3, "call-rate-per-sample-imiss-and-per-snp-lmiss"]], "Plot histograms of call rate per sample": [[3, "plot-histograms-of-call-rate-per-sample"]], "Plot histograms of call rate per snp": [[3, "plot-histograms-of-call-rate-per-snp"]], "Plot histograms of call rate per sample after filtered SNPs": [[3, "plot-histograms-of-call-rate-per-sample-after-filtered-snps"]], "2. Check MAF (SNP-level)": [[3, "check-maf-snp-level"]], "Plot MAF histogram": [[3, "plot-maf-histogram"]], "3. Check sample heterozygosity (Sample-level)": [[3, "check-sample-heterozygosity-sample-level"]], "Plot sample call rate vs heterozygosity": [[3, "plot-sample-call-rate-vs-heterozygosity"]], "4. Check sex discordance (Sample-level)": [[3, "check-sex-discordance-sample-level"]], "Plot reported sex vs F value (X heterozygosity level).": [[3, "plot-reported-sex-vs-f-value-x-heterozygosity-level"]], "Check Y chr counts also if there are strange samples": [[3, "check-y-chr-counts-also-if-there-are-strange-samples"]], "Plot y chromosome counts": [[3, "plot-y-chromosome-counts"]], "5. Check relatedness (Sample-level)": [[3, "check-relatedness-sample-level"]], "LD pruning": [[3, "ld-pruning"]], "Calculate IBD - identity by descent": [[3, "calculate-ibd-identity-by-descent"]], "6. Hardy\u2013Weinberg equilibrium (SNP level)": [[3, "hardy-weinberg-equilibrium-snp-level"]], "7. Sample QC: PCA": [[3, "sample-qc-pca"]], "Last step:": [[3, "last-step"]], "Final remarks": [[3, "final-remarks"]], "Author": [[3, "author"]], "References": [[3, "references"]]}, "indexentries": {}})
>>>>>>> ab67cd767b8a60c6441619cc9e7b4939d753b40b
