Search.setIndex({"docnames": ["genomestudio", "gwide", "index", "qualitycontrol", "setup"], "filenames": ["genomestudio.md", "gwide.md", "index.md", "qualitycontrol.md", "setup.md"], "titles": ["From raw to plink", "GWAS software", "SNP/GWAS workflow documentation!", "1. Check call rate per SNP and per sample (both SNP and Sample-level)", "Set-up"], "terms": {"index": [], "modul": [], "search": [], "page": [], "A": [2, 3], "flexibl": [], "r": [3, 4], "packag": 4, "process": 3, "data": [2, 3], "from": [3, 4], "differ": 3, "plate": [], "reader": [], "machin": [], "The": [2, 3], "main": [], "function": [3, 4], "i": [0, 2, 3, 4], "provid": 3, "simpl": [], "method": 3, "can": 3, "raw": [], "micropl": [], "like": [], "spectramax": [], "fluorstar": [], "multiscango": [], "appli": 3, "thi": [0, 2, 3, 4], "gener": [0, 3], "tidi": [], "tibbl": [], "object": [], "combin": [], "layout": [], "file": [2, 3, 4], "growth": [], "rate": [], "ar": [2, 3], "calcul": 3, "plot": [], "over": [], "frame": 3, "critic": [], "point": 0, "sever": [], "research": 3, "line": [], "perform": [2, 3, 4], "experi": [], "us": [0, 3], "mainli": [], "due": 3, "time": [], "take": 3, "clean": [], "sort": [], "format": [2, 3], "also": 3, "featur": [], "allow": [], "detect": 3, "which": 3, "well": [], "present": [], "faulti": [], "e": [2, 3], "g": [2, 3], "fill": [], "correctli": [], "anoth": [], "veri": [0, 3], "toctre": [], "section": [], "api": [], "genotyp": [2, 3], "da": [], "workflow": [], "cover": [], "intens": [], "imput": 3, "gwa": 3, "analysi": 3, "note": [], "project": [], "under": [1, 3], "activ": [], "develop": 1, "plink": [2, 3, 4], "step": [2, 4], "qc": [0, 2, 4], "overview": [], "qualiti": 3, "control": 3, "genom": [2, 3], "wide": 3, "associ": 3, "studi": [2, 3], "read": 3, "intes": [], "hand": [], "studio": [], "Will": 4, "need": [2, 3, 4], "gnomestudio": 4, "plink1": [2, 4], "9": [2, 3, 4], "we": [0, 3], "list": [2, 3], "go": [], "defin": 3, "insid": [], "runplink": [3, 4], "plinkopt": 4, "system": [3, 4], "past": 4, "most": [2, 4], "so": [2, 3, 4], "ha": [2, 4], "instal": [2, 4], "let": [2, 3, 4], "": [2, 3, 4], "prepar": [2, 4], "environ": [2, 4], "upload": [2, 4], "librari": [2, 4], "creat": [3, 4], "run": 4, "directori": 4, "where": [3, 4], "ex": 4, "you": [2, 3, 4], "might": [3, 4], "chang": [3, 4], "depend": [3, 4], "wai": 4, "plan": [3, 4], "dplyr": 4, "ggplot2": 4, "ggrepel": 4, "describ": 2, "either": 2, "ped": [0, 2, 3], "map": [0, 2, 3], "bed": [2, 3], "contain": 2, "inform": [2, 3], "each": [2, 3], "particip": [2, 3], "includ": 2, "famili": [2, 3], "id": 2, "father": 2, "mother": 2, "sex": 2, "phenotyp": [2, 3], "full": 2, "type": 2, "here": 2, "bi": 2, "allel": [2, 3], "onli": [2, 3], "two": [2, 3], "nucleotid": 2, "observ": [2, 3], "ani": 2, "given": [2, 3], "across": [2, 3], "code": [2, 3], "pair": [2, 3], "c": [2, 3], "t": [2, 3], "notabl": 2, "order": 2, "non": 2, "sens": 2, "first": [2, 3], "necessarili": 2, "same": [2, 3], "chromosom": [2, 3], "row": 2, "correspond": [2, 3], "chr": [2, 3], "coordin": 2, "bppo": 2, "base": [2, 3], "current": [2, 3], "build": [2, 3], "proport": 3, "individu": 3, "miss": 3, "out": 3, "file_call_r": 3, "make": 3, "easier": 3, "cat": 3, "tr": 3, "call_rate_sampl": 3, "tab": 3, "call_rate_snp": 3, "histogram": [], "load": 3, "delim": 3, "select": [0, 3], "x": 0, "n": 3, "filter": [], "our": 3, "0": 3, "n_filtered_out": 3, "sum": 3, "f_miss": 3, "ggplot": 3, "ae": 3, "100": 3, "geom_histogram": 3, "bin": 3, "50": 3, "theme_bw": 3, "ylab": 3, "number": 3, "xlab": 3, "geom_vlin": 3, "xintercept": 3, "90": 3, "color": 3, "red": 3, "linetyp": 3, "dash": 3, "annot": 3, "geom": 3, "text": 3, "70": 3, "y": [0, 3], "100000": 3, "label": 3, "paste0": 3, "exclud": 3, "size": 3, "4": [], "again": 3, "geno": 3, "recod": 3, "file_filtered_snp": 3, "file_call_rate_filtered_snp": 3, "file_call_rate_filtered_snps_imiss": 3, "call_rate_samples_after_filt": 3, "after": [], "iter": 3, "help": 3, "decid": 3, "threshold": 3, "choos": 3, "low": 3, "usual": 3, "vari": 3, "95": 3, "98": 3, "200": 3, "minor": 3, "frequenc": 3, "larg": 3, "degre": 3, "homogen": 3, "result": 3, "inadequ": 3, "power": 3, "infer": 3, "statist": 3, "signific": 3, "relationship": 3, "between": 3, "trait": 3, "occur": 3, "when": 3, "have": 3, "small": 3, "major": 3, "copi": 3, "5": 3, "set": 3, "freq": 3, "file_maf": 3, "frq": 3, "file_maf_frq": 3, "maf_check": 3, "maf_01": 3, "01": 3, "maf_05": 3, "05": 3, "grei": 3, "02": 3, "angl": 3, "3": [], "043": 3, "For": 3, "recommend": 3, "refer": [], "presenc": 3, "within": 3, "an": 3, "outlier": 3, "show": 3, "excess": 3, "deplet": 3, "heterozygot": 3, "mai": 3, "contamin": 3, "issu": 3, "dure": 3, "should": [0, 3], "do": 3, "alreadi": 3, "high": 3, "het": 3, "file_het": 3, "het_level": 3, "more": 3, "than": 3, "standard": 3, "deviat": 3, "awai": 3, "mean": 3, "lifelin": 3, "4sd": 3, "sample_missing_rate_het": 3, "left_join": 3, "iid": 3, "renam": 3, "heter_r": 3, "f": [], "std": 3, "sd": 3, "std_list": 3, "geom_point": 3, "blue": 3, "geom_hlin": 3, "yintercept": 3, "col": 3, "6": [], "ggtitl": 3, "missing": 3, "mix": 3, "up": 3, "investig": 3, "look": 3, "mismatch": 3, "compar": 3, "nb": 3, "howev": 3, "doe": 3, "reliabl": 3, "abnorm": 3, "mind": 3, "file_snp_filt": 3, "now": 3, "sexcheck": 3, "file_snp_filtered_sexcheck": 3, "taken": 3, "plink_sex_filt": 3, "mutat": 3, "pedsex": 3, "if_els": 3, "femal": 3, "male": 3, "geom_jitt": 3, "ifels": 3, "geom_text_repel": 3, "hjust": 3, "vjust": 3, "mark": 3, "problemat": 3, "avail": 3, "advis": 3, "It": 3, "import": 3, "ass": 3, "visual": 3, "ycount": 3, "file_snp_filtered_ycount": 3, "file_snp_filtered_ycount_sexcheck": 3, "plink_sex_filtered_ycount": 3, "clearli": 3, "separ": 3, "axi": 3, "3000": 3, "count": 3, "strang": 3, "related": 3, "popul": 3, "cohort": 3, "often": 3, "limit": 3, "unrel": 3, "linear": 3, "model": 3, "approach": 3, "assum": 3, "independ": 3, "importantli": 3, "region": 3, "hospit": 3, "complex": 3, "diseas": 3, "recruit": 3, "unintention": 3, "common": 3, "measur": 3, "duplic": 3, "ident": 3, "descent": 3, "ibd": 3, "kinship": 3, "coeffici": 3, "greater": 3, "1875": 3, "suggest": 3, "mixtur": 3, "typic": 3, "relat": 3, "lower": 3, "remov": 3, "ld": 3, "prune": 3, "linkag": 3, "disequilibrium": 3, "could": 3, "some": 3, "other": 3, "one": 3, "elimin": 3, "redund": 3, "dimens": 3, "reduct": 3, "commonli": 3, "prior": 3, "indep": 3, "pairwis": 3, "keep": 3, "your": 3, "prunedset": 3, "recent": 3, "share": 3, "ancestri": 3, "z": 3, "pruned_ibd": 3, "zcat": 3, "gz": 3, "pi_hat": 3, "column": 3, "output": 3, "tabl": 3, "ibd_high_relat": 3, "print": 3, "violat": 3, "hwe": 3, "indic": 3, "substructur": 3, "occurr": 3, "error": 3, "while": 3, "thei": 3, "alwai": 3, "distinguish": 3, "practic": 3, "If": 3, "case": 3, "statu": 3, "departur": 3, "\ud835\udf122": 3, "good": 3, "fit": 3, "test": 3, "expect": 3, "doesn": 3, "reformat": 3, "add": 3, "consid": 3, "p": 3, "10e": 3, "only_control": 3, "controls_hardi": 3, "controls_hardy_hw": 3, "plink_controls_hw": 3, "unaff": 3, "identif": 3, "diverg": 3, "requir": 3, "dataset": 3, "divers": 3, "ethnic": 3, "hgdp": 3, "hapmap": 3, "exclus": 3, "question": 3, "all": 3, "filterd": 3, "delet": 3, "ind": 3, "fail": 3, "txt": 3, "qced": 3, "extra": 3, "swap": 3, "arrai": 3, "rna": 3, "seq": 3, "care": 3, "creation": 3, "michigan": 3, "server": 3, "vcf": 3, "bcftool": 3, "view": 3, "r2": 3, "oz": 3, "input_fil": 3, "output_fil": 3, "These": 3, "updat": 3, "document": 3, "turner": 3, "stephen": 3, "et": 3, "al": 3, "procedur": 3, "protocol": 3, "human": 3, "genet": 3, "68": 3, "2011": 3, "19": 3, "reed": 3, "eric": 3, "guid": 3, "post": 3, "analyt": 3, "interrog": 3, "medicin": 3, "34": 3, "28": 3, "2015": 3, "3769": 3, "3792": 3, "anderson": 3, "carl": 3, "natur": 3, "2010": 3, "1564": 3, "1573": 3, "To": 0, "easi": 0, "work": 0, "imag": 0, "want": 0, "autosom": 0, "chromosoma": 0, "improv": 0, "At": 0, "abl": 0, "continu": 0, "martin": 0, "banchero": 0, "rui": 0, "marcelo": 0, "tatiana": 3, "karp": 3}, "objects": {}, "objtypes": {}, "objnames": {}, "titleterms": {"welcom": [], "genotypeda": [], "umcg": [], "griac": [], "": [], "document": 2, "indic": [], "tabl": [], "mpxtractor": [], "content": [], "gwa": [1, 2], "raw": 0, "data": 0, "process": [], "qualiti": 2, "control": 2, "workflow": 2, "hand": [], "genom": [], "studio": [], "gener": [], "plink": 0, "file": 0, "snp": [2, 3], "set": 4, "up": 4, "softwar": [1, 4], "arrai": 2, "protocol": 2, "1": 3, "check": 3, "call": 3, "rate": 3, "per": 3, "sampl": 3, "both": 3, "level": 3, "imiss": 3, "lmiss": 3, "plot": 3, "histogram": 3, "exclud": [], "90": [], "again": [], "after": 3, "filter": 3, "2": 3, "maf": 3, "3": 3, "heterozygos": 3, "v": 3, "4": 3, "sex": 3, "discord": 3, "report": 3, "f": 3, "valu": 3, "x": 3, "y": [], "chr": [], "count": [], "also": [], "ar": [], "strang": [], "chromosom": [], "6": 3, "hardi": 3, "weinberg": 3, "equilibrium": 3, "7": 3, "qc": 3, "pca": 3, "last": 3, "step": 3, "final": 3, "remark": 3, "refer": 3, "from": 0, "load": 0, "re": 0, "cluster": 0, "genomestudio": 0, "output": 0, "author": [0, 3]}, "envversion": {"sphinx.domains.c": 3, "sphinx.domains.changeset": 1, "sphinx.domains.citation": 1, "sphinx.domains.cpp": 9, "sphinx.domains.index": 1, "sphinx.domains.javascript": 3, "sphinx.domains.math": 2, "sphinx.domains.python": 4, "sphinx.domains.rst": 2, "sphinx.domains.std": 2, "sphinx": 60}, "alltitles": {"Set-up": [[4, "set-up"]], "Software": [[4, "software"]], "1. Check call rate per SNP and per sample (both SNP and Sample-level)": [[3, "check-call-rate-per-snp-and-per-sample-both-snp-and-sample-level"]], "Call rate per sample (imiss) and per SNP (lmiss)": [[3, "call-rate-per-sample-imiss-and-per-snp-lmiss"]], "Plot histograms of call rate per sample": [[3, "plot-histograms-of-call-rate-per-sample"]], "Plot histograms of call rate per snp": [[3, "plot-histograms-of-call-rate-per-snp"]], "Plot histograms of call rate per sample after filtered SNPs": [[3, "plot-histograms-of-call-rate-per-sample-after-filtered-snps"]], "2. Check MAF (SNP-level)": [[3, "check-maf-snp-level"]], "Plot MAF histogram": [[3, "plot-maf-histogram"]], "3. Check sample heterozygosity (Sample-level)": [[3, "check-sample-heterozygosity-sample-level"]], "Plot sample call rate vs heterozygosity": [[3, "plot-sample-call-rate-vs-heterozygosity"]], "4. Check sex discordance (Sample-level)": [[3, "check-sex-discordance-sample-level"]], "Plot reported sex vs F value (X heterozygosity level).": [[3, "plot-reported-sex-vs-f-value-x-heterozygosity-level"]], "6. Hardy\u2013Weinberg equilibrium (SNP level)": [[3, "hardy-weinberg-equilibrium-snp-level"]], "7. Sample QC: PCA": [[3, "sample-qc-pca"]], "Last step:": [[3, "last-step"]], "Final remarks": [[3, "final-remarks"]], "Author": [[3, "author"]], "References": [[3, "references"]], "SNP/GWAS workflow documentation!": [[2, "snp-gwas-workflow-documentation"]], "SNP array quality control protocol": [[2, "snp-array-quality-control-protocol"]], "GWAS software": [[1, "gwas-software"]], "From raw to plink": [[0, "from-raw-to-plink"]], "Load data": [[0, "load-data"]], "Re-cluster": [[0, "re-cluster"]], "genomestudio output files": [[0, "genomestudio-output-files"]], "Authors": [[0, "authors"]]}, "indexentries": {}})