Search.setIndex({"docnames": ["Check_MAF", "Check_call_rate_per_SNP_per_sample", "Check_relatedness", "Check_sample_heterozygosity", "Check_sex_discordance", "Final_remarks", "GWAS", "Hardy--Weinberg_equilibrium", "PCA", "Processing_raw_data", "QC_overview", "index"], "filenames": ["Check_MAF.rst", "Check_call_rate_per_SNP_per_sample.rst", "Check_relatedness.rst", "Check_sample_heterozygosity.rst", "Check_sex_discordance.rst", "Final_remarks.rst", "GWAS.rst", "Hardy--Weinberg_equilibrium.rst", "PCA.rst", "Processing_raw_data.rst", "QC_overview.rst", "index.rst"], "titles": ["2. Check MAF (SNP-level)", "1. Check call rate per SNP and per sample (both SNP and Sample-level)", "5. Check relatedness (Sample-level)", "3. Check sample heterozygosity (Sample-level)", "4. Check sex discordance (Sample-level)", "Final remarks", "GWAS", "6. Hardy\u2013Weinberg equilibrium (SNP level)", "7. Sample QC: PCA (Sample-level)", "Raw data processing", "<strong>SNP array quality control protocol</strong>", "Welcome to genotype data analysis UMCG-GRIAC documentation!"], "terms": {"index": [], "modul": [], "search": [], "page": [], "A": [0, 2, 10], "flexibl": [], "r": 1, "packag": 10, "process": [3, 11], "data": [1, 2, 10], "from": [1, 2, 3, 4, 5, 7, 9, 10, 11], "differ": [], "plate": [], "reader": [], "machin": [], "The": [1, 10, 11], "main": [], "function": [5, 10], "i": [0, 1, 2, 4, 5, 6, 7, 9, 10, 11], "provid": [2, 7], "simpl": [], "method": 4, "can": [0, 1, 2, 5, 7], "raw": [], "micropl": [], "like": [], "spectramax": [], "fluorstar": [], "multiscango": [], "appli": 2, "thi": [0, 1, 2, 4, 6, 7, 8, 9, 10, 11], "gener": [0, 2, 7], "tidi": [], "tibbl": [], "object": [], "combin": [], "layout": [], "file": [0, 1, 2, 3, 4, 5, 7, 10], "growth": [], "rate": [2, 4], "ar": [2, 5, 7, 10], "calcul": [1, 3], "plot": 4, "over": [], "frame": 1, "critic": [], "point": [], "sever": [], "research": 8, "line": [], "perform": [2, 5, 10], "experi": [], "us": [2, 3, 4, 5, 7, 9], "mainli": [], "due": 3, "time": [], "take": 5, "clean": [], "sort": [], "format": [1, 2, 10], "also": [], "featur": [], "allow": [], "detect": 4, "which": [1, 7], "well": [], "present": [], "faulti": [], "e": [2, 10], "g": [2, 10], "fill": [], "correctli": [], "anoth": [], "veri": 0, "toctre": [], "section": [], "api": [], "genotyp": [2, 3, 4, 7, 9, 10], "da": [], "workflow": [6, 9, 11], "cover": [6, 9, 11], "intens": [9, 11], "imput": [5, 11], "gwa": [2, 11], "analysi": [2, 6], "note": [], "project": [6, 9, 11], "under": [0, 6, 9, 11], "activ": [6, 9, 11], "develop": [6, 9, 11], "plink": [2, 4, 5, 10], "step": [2, 4, 5, 8, 10], "qc": [5, 10], "overview": [], "qualiti": 1, "control": 7, "genom": [2, 5, 10], "wide": [], "associ": 7, "studi": [0, 1, 2, 10], "read": [0, 1, 3, 4, 7, 9], "intes": 9, "hand": [], "studio": [], "minor": 0, "allel": [0, 3, 10], "frequenc": 0, "larg": [0, 2], "degre": 0, "homogen": 0, "given": [0, 1, 3, 7, 10], "across": [0, 2, 3, 10], "particip": [0, 10], "result": 0, "inadequ": 0, "power": 0, "infer": 0, "statist": [0, 7], "signific": 0, "relationship": 0, "between": [0, 2, 7], "trait": 0, "occur": 0, "when": [0, 4], "we": [0, 2, 4], "have": [0, 7], "small": 0, "so": [0, 10], "major": 0, "individu": [0, 1, 2, 3, 8], "two": [0, 3, 10], "copi": 0, "threshold": [0, 1, 2, 7], "vari": [0, 1, 2, 7], "1": [0, 3, 4], "5": [0, 3, 7], "sampl": [0, 5], "set": 0, "advic": [0, 2, 3, 4], "For": [0, 4, 5], "size": [0, 1], "recommend": 0, "exclud": [0, 2, 3, 4], "0": [0, 1, 2, 3, 4, 5], "05": [0, 5], "defin": 1, "proport": [1, 3], "correspond": [1, 10], "inform": [1, 4, 10], "miss": [1, 3], "imiss": 1, "lmiss": 1, "runplink": [0, 1, 2, 3, 4, 7, 10], "ped": [0, 1, 2, 3, 4, 7, 10], "map": [0, 1, 2, 3, 4, 7, 10], "out": [0, 1, 2, 3, 4, 7], "file_call_r": 1, "chang": [1, 2, 10], "system": [0, 1, 2, 3, 4, 7, 10], "cat": [0, 1, 3, 4, 7], "tr": [0, 1, 2, 3, 4, 7], "": [0, 1, 2, 3, 4, 7, 10], "t": [0, 1, 2, 3, 4, 7, 10], "call_rate_snp": 1, "tab": [0, 1, 2, 3, 4, 7], "load": 1, "delim": [0, 1, 3, 4, 7], "select": [0, 1, 2, 3, 4, 7], "x": [0, 1, 3, 4, 7], "call_rate_sampl": 1, "popul": [2, 7], "base": [2, 7, 10], "cohort": 2, "often": 2, "limit": 2, "unrel": 2, "linear": 2, "model": 2, "approach": 2, "assum": [2, 7], "independ": 2, "importantli": 2, "region": 2, "hospit": 2, "complex": 2, "diseas": 2, "same": [2, 4, 10], "famili": [2, 10], "recruit": 2, "unintention": 2, "common": [2, 7], "measur": [2, 7], "duplic": 2, "pair": [2, 10], "an": [2, 3, 7], "kinship": 2, "coeffici": 2, "greater": 2, "than": [2, 3], "1875": 2, "mai": [2, 3, 7], "suggest": 2, "mixtur": 2, "typic": 2, "relat": 2, "lower": 2, "call": [2, 4], "remov": [2, 7], "first": [2, 10], "need": [2, 4, 5, 10], "linkag": 2, "disequilibrium": 2, "valu": [2, 4, 7], "7": 2, "could": 2, "some": 2, "2": [2, 3, 4], "snp": [2, 3, 4, 5], "high": [2, 3], "each": [2, 3, 4, 10], "other": 2, "one": 2, "elimin": 2, "redund": 2, "dimens": 2, "reduct": 2, "commonli": 2, "prior": 2, "both": 2, "pca": 2, "refer": [3, 5, 8], "presenc": [3, 7], "within": 3, "outlier": 3, "show": 3, "excess": 3, "deplet": 3, "heterozygot": 3, "contamin": 3, "issu": 3, "dure": 3, "should": [4, 7], "do": 3, "alreadi": [3, 4], "onli": [2, 3, 7, 10], "advis": [], "more": 3, "standard": 3, "deviat": 3, "awai": 3, "mean": 3, "lifelin": 3, "4sd": 3, "mix": 4, "up": 4, "investig": 4, "look": 4, "mismatch": 4, "where": [4, 10], "compar": [4, 5], "chromosom": [4, 10], "y": [0, 1, 3], "nb": 4, "howev": 4, "doe": [4, 7], "reliabl": 4, "abnorm": 4, "low": [1, 4], "check": [5, 7], "phenotyp": [7, 10], "If": 7, "doesn": 7, "reformat": 7, "add": 7, "code": [2, 7, 10], "hwe": 7, "test": 7, "consid": [3, 4, 5, 7], "p": 7, "10e": 7, "describ": [10, 11], "you": [1, 5, 10], "either": 10, "bed": 10, "contain": 10, "includ": 10, "id": 10, "father": 10, "mother": 10, "sex": 10, "full": 10, "type": 10, "here": 10, "bi": 10, "nucleotid": 10, "observ": [7, 10], "ani": 10, "c": [3, 10], "notabl": 10, "order": 10, "non": 10, "sens": 10, "list": 10, "necessarili": 10, "row": 10, "chr": 10, "coordin": 10, "bppo": 10, "current": [5, 10], "build": [5, 10], "most": 10, "plink1": 10, "9": 10, "ha": [4, 10], "instal": 10, "let": [2, 4, 10], "prepar": 10, "environ": 10, "upload": 10, "librari": 10, "creat": [7, 10], "run": 10, "directori": 10, "ex": 10, "plinkopt": 10, "past": 10, "might": [5, 8, 10], "depend": [1, 8, 10], "wai": 10, "plan": [5, 10], "rational": 11, "follow": 11, "figur": 11, "make": 1, "easier": 1, "n": 1, "filter": [1, 2, 7], "n_filtered_out": 1, "sum": [0, 1], "f_miss": [1, 3], "ggplot": [0, 1, 3, 4], "ae": [0, 1, 3, 4], "100": 1, "geom_histogram": [0, 1], "bin": 1, "50": [1, 2], "theme_bw": [0, 1, 3, 4], "ylab": [0, 1, 3, 4], "number": 1, "xlab": [0, 1, 3, 4], "geom_vlin": [0, 1, 3], "xintercept": [0, 1, 3], "90": 0, "color": [0, 1, 3, 4], "red": [0, 1, 3, 4], "linetyp": [0, 1, 3], "dash": [0, 1, 3], "annot": [0, 1], "geom": [0, 1], "text": [0, 1], "70": 1, "200": 1, "label": [0, 1, 4], "paste0": [0, 1], "4": [1, 3, 5], "100000": [0, 1], "geno": 1, "recod": [1, 2, 4, 5], "file_filtered_snp": 1, "caption": [], "file_call_rate_filtered_snp": 1, "file_call_rate_filtered_snps_imiss": 1, "call_rate_samples_after_filt": [1, 3], "after": [1, 5], "iter": 1, "help": 1, "decid": 1, "choos": 1, "usual": 1, "95": 1, "98": 1, "freq": 0, "file_maf": 0, "frq": 0, "file_maf_frq": 0, "maf_check": 0, "maf_01": 0, "01": 0, "maf_05": 0, "grei": [0, 3], "02": 0, "angl": 0, "3": [0, 5], "043": 0, "het": 3, "file_het": 3, "het_level": 3, "sample_missing_rate_het": 3, "left_join": 3, "iid": [3, 4], "renam": 3, "heter_r": 3, "f": [3, 4], "std": 3, "sd": 3, "std_list": 3, "geom_point": 3, "blue": [3, 4], "geom_hlin": 3, "yintercept": 3, "col": 3, "6": 3, "ggtitl": 3, "missing": 3, "fot": 4, "record": 4, "otherwis": 4, "metadata": 4, "mind": 4, "file_snp_filt": 4, "now": 4, "sexcheck": 4, "file_snp_filtered_sexcheck": 4, "report": 4, "v": [4, 5], "heterozygos": 4, "plink_sex_filt": 4, "mutat": 4, "pedsex": 4, "if_els": 4, "femal": 4, "male": 4, "geom_jitt": 4, "ifels": 4, "geom_text_repel": 4, "hjust": 4, "vjust": 4, "mark": 4, "problemat": 4, "avail": [4, 7], "It": 4, "import": 4, "ass": 4, "visual": 4, "count": [], "strang": [], "ycount": 4, "file_snp_filtered_ycount": [], "file_snp_filtered_ycount_sexcheck": [], "plink_sex_filtered_ycount": 4, "3000": 4, "clearli": 4, "separ": [4, 7], "axi": 4, "violat": 7, "indic": 7, "substructur": 7, "occurr": 7, "error": 7, "while": 7, "thei": 7, "alwai": 7, "distinguish": 7, "practic": 7, "case": 7, "statu": 7, "departur": 7, "\ud835\udf122": 7, "good": 7, "fit": 7, "expect": 7, "only_control": 7, "controls_hardi": 7, "controls_hardy_hw": 7, "plink_controls_hw": 7, "unaff": 7, "prunedset": 2, "z": 2, "pruned_ibd": 2, "zcat": 2, "gz": 2, "pi_hat": 2, "column": 2, "output": 2, "ibd_high_relat": 2, "print": 2, "tabl": 2, "indep": 2, "pairwis": 2, "keep": 2, "your": 2, "extra": 5, "swap": 5, "arrai": 5, "rna": 5, "seq": 5, "care": 5, "creation": 5, "michigan": 5, "server": 5, "vcf": 5, "bcftool": 5, "view": 5, "r2": 5, "maf": 5, "oz": 5, "input_fil": 5, "output_fil": 5, "These": 5, "updat": 5, "document": 5, "identif": 8, "diverg": 8, "ancestri": 8, "requir": 8, "dataset": 8, "divers": 8, "ethnic": 8, "hgdp": 8, "hapmap": 8, "exclus": 8, "question": 8, "dplyr": 10, "ggplot2": 10, "ggrepel": 10}, "objects": {}, "objtypes": {}, "objnames": {}, "titleterms": {"welcom": 11, "genotypeda": [], "umcg": 11, "griac": 11, "": [], "document": 11, "indic": [], "tabl": [], "mpxtractor": [], "content": [], "gwa": 6, "raw": 9, "data": [9, 11], "process": 9, "qualiti": 10, "control": 10, "workflow": [], "hand": 9, "genom": 9, "studio": 9, "gener": 9, "plink": 9, "file": 9, "2": 0, "check": [0, 1, 2, 3, 4], "maf": 0, "snp": [0, 1, 7, 10], "level": [0, 1, 2, 3, 4, 7, 8], "plot": [0, 1, 3], "histogram": [0, 1], "1": 1, "call": [1, 3], "rate": [1, 3], "per": 1, "sampl": [1, 2, 3, 4, 8], "both": 1, "5": 2, "related": 2, "ld": 2, "prune": 2, "calcul": 2, "ibd": 2, "ident": 2, "descent": 2, "degre": 2, "recent": 2, "share": 2, "ancestri": 2, "3": 3, "heterozygos": 3, "v": 3, "4": 4, "sex": 4, "discord": 4, "6": 7, "hardi": 7, "weinberg": 7, "equilibrium": 7, "arrai": 10, "protocol": 10, "introduct": [], "genotyp": 11, "analysi": 11, "tatiana": [], "karp": [], "exclud": 1, "90": 1, "again": 1, "y": 4, "chr": 4, "count": 4, "also": 4, "ar": 4, "strang": 4, "final": 5, "remark": 5, "7": 8, "qc": 8, "pca": 8}, "envversion": {"sphinx.domains.c": 3, "sphinx.domains.changeset": 1, "sphinx.domains.citation": 1, "sphinx.domains.cpp": 9, "sphinx.domains.index": 1, "sphinx.domains.javascript": 3, "sphinx.domains.math": 2, "sphinx.domains.python": 4, "sphinx.domains.rst": 2, "sphinx.domains.std": 2, "sphinx": 60}, "alltitles": {"GWAS": [[6, "gwas"]], "Raw data processing": [[9, "raw-data-processing"]], "Hands on genome studio": [[9, "hands-on-genome-studio"]], "Generate plink files": [[9, "generate-plink-files"]], "Welcome to genotype data analysis UMCG-GRIAC documentation!": [[11, "welcome-to-genotype-data-analysis-umcg-griac-documentation"]], "1. Check call rate per SNP and per sample (both SNP and Sample-level)": [[1, "check-call-rate-per-snp-and-per-sample-both-snp-and-sample-level"]], "Plot histograms of call rate per sample": [[1, "plot-histograms-of-call-rate-per-sample"]], "Plot histograms of call rate per snp": [[1, "plot-histograms-of-call-rate-per-snp"]], "Exclude SNPs with call rate < 90% and check again sample call rate": [[1, "exclude-snps-with-call-rate-90-and-check-again-sample-call-rate"]], "2. Check MAF (SNP-level)": [[0, "check-maf-snp-level"]], "Plot MAF histogram": [[0, "plot-maf-histogram"]], "3. Check sample heterozygosity (Sample-level)": [[3, "check-sample-heterozygosity-sample-level"]], "Plot sample call rate vs heterozygosity": [[3, "plot-sample-call-rate-vs-heterozygosity"]], "6. Hardy\u2013Weinberg equilibrium (SNP level)": [[7, "hardy-weinberg-equilibrium-snp-level"]], "4. Check sex discordance (Sample-level)": [[4, "check-sex-discordance-sample-level"]], "Check Y chr counts also if there are strange samples": [[4, "check-y-chr-counts-also-if-there-are-strange-samples"]], "5. Check relatedness (Sample-level)": [[2, "check-relatedness-sample-level"]], "LD pruning": [[2, "ld-pruning"]], "Calculate IBD - identity by descent (the degree of recent shared ancestry)": [[2, "calculate-ibd-identity-by-descent-the-degree-of-recent-shared-ancestry"]], "Final remarks": [[5, "final-remarks"]], "7. Sample QC: PCA (Sample-level)": [[8, "sample-qc-pca-sample-level"]], "SNP array quality control protocol": [[10, "snp-array-quality-control-protocol"]]}, "indexentries": {}})